### PR TITLE
Prepare for prioritize issues & update .gitignore for goland users

### PR DESCRIPTION
**Pull request by BetaCat0**
Prepare for Prioritize issues & update .gitignore for goland users.
Target on the [Issue](https://github.com/go-gitea/gitea/issues/2616)

Signed-off-by: BetaCat0 <outman99@hotmail.com>

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

File: .gitignore [hunk 1]
```diff
@@ -24,3 +24,4 @@ _testmain.go
 *.prof
 
 coverage.out
+.idea
```
Comment (appleboy) [inline .gitignore]: Please revert this change. You can add to global ignore file.

**Review by appleboy — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (BetaCat0) [reply .gitignore]: fixed :)

**Review by appleboy — APPROVED** [review]
